docs/getting-started/user-guides/using-paymetheus.md

```diff
@@ -0,0 +1,151 @@
+# **The Paymetheus Windows Wallet** #
+This guide assumes you have already set up a Paymetheus wallet using [this guide](paymetheus.md).
+
+---
+
+## **Overview** ##
+The overview tab gives a quick summary of your total DCR (spendable and locked), number of accounts and transactions as well as a list 
+of recent account activity.  
+
+![Overview tab](/img/Paymetheus-overview.png)  
+
+---
+
+
+## **Accounts  ** ##
+The accounts tab shows you the accounts in your wallet and allows you to add new ones. Accounts in Decred work just like bank accounts. 
+They allow you to keep seperate records of your DCR. This feature is most 
+useful for those who run businesses and wish to keep seperate accounts for 
+tax records for example. Transferring DCR across accounts will create a 
+transaction on the blockchain.  
+
+![Overview tab](/img/Paymetheus-accounts.png)  
+
+---
+
+
+## **Scripts** ##
+Currently only used for PoS mining in a pool. If you sign up for a pool, 
+you will be asked to import a script. This is where you will add it.
+See the Purchase Tickets section below for more information.  
+
+![Scripts tab](/img/Paymetheus-import-script.png)  
+
+---
+
+
+## **Create Transaction** ##
+This tab is how you send funds to another address. Simply copy the receivers
+address into the text box and type in the amount of Decred you wish to send.
+The estimated fee will be listed as well. You can click the '+' button to 
+send Decred to multiple accounts in the one transaction if you wish.  
+
+![Create transaction tab](/img/Paymetheus-send.png)  
+
+---
+
+
+## ** Purchase Tickets tab** ##
+
+Stake pools are also usable with Paymetheus by using its manual ticket
+purchasing features.  
+
+> To join the pool, provide a public key address which can be used to generate a
+> 1-of-2 multisignature script. The multisignature script will be generated by
+> the pool and returned to you along with a P2SH address to give voting rights to.  
+
+Don't worry if you didn't understand that quote. What it means is that you create
+an address that can be accessed by two wallets. Only one wallet needs to be available
+to use the address. This means that the pool can vote on your behalf and you can vote
+using your own wallet if the pool stops working.  
+
+It DOES NOT give the pool access to your funds. All you are doing is granting voting
+rights to the wallet. This is why you provide a ticket address so the network knows
```

Comment:
I've removed the bit about tickets and voting access because it confused me! Have a look at the new copy when it comes through and see if it's ok. If you want something about it in there, I might need someone else to provide me with a few sentences since I don't really know enough about how that works.

Given that the next release is only a week away, hopefully this will be good.